frontend/src/api/common/api-request-builder/test-utils/expect-fetch.ts

```diff
@@ -25,8 +30,20 @@ export const expectFetch = (expectedUrl: string, requestStatusCode: number, expe
     })
     return Promise.resolve(
       Mock.of<Response>({
-        status: requestStatusCode
+        status: requestStatusCode,
+        statusText: mapCodeToText(requestStatusCode),
+        json: jest.fn(() => (responseBody ? Promise.resolve(responseBody) : Promise.reject()))
       })
     )
   })
 }
+const mapCodeToText = (code: number): string => {
+  switch (code) {
+    case 400:
+      return 'bad_request'
+    case 401:
+      return 'forbidden'
+    default:
+      return 'unknown code'
+  }
+}
```

Comment:
Do we want to maintain this list here?

Reply:
its only relevant for tests

Reply:
Still, this looks a bit misplaced and could be extracted into an own file.

Reply:
why? it's only used in the fetch mock to simulate the "statusText" property.